bindata/v4.1.0/kube-apiserver/check-endpoints-clusterrolebinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: check-endpoints
```

Comment:
`system:openshift:operator?:kube-apiserver-check-endpoints`

Looks to see what fits best in operator.